openshift/e2e-common.sh

```diff
@@ -195,15 +201,6 @@ function prepare_knative_serving_tests_nightly {
   export GATEWAY_OVERRIDE=kourier
   export GATEWAY_NAMESPACE_OVERRIDE="$SERVING_INGRESS_NAMESPACE"
   export INGRESS_CLASS=kourier.ingress.networking.knative.dev
-
-  if [[ ${ENABLE_INTERNAL_TLS} == "true" ]]; then
-    # Deploy CA cert for testing TLS with cluster-local gateway
-    yq read --doc 0 ./test/config/tls/cert-secret.yaml | oc apply -f -
-    # This needs to match the name of Secret in test/config/tls/cert-secret.yaml
-    export CA_CERT=ca-cert
-    # This needs to match $san from test/config/tls/generate.sh
-    export SERVER_NAME=knative.dev
-  fi
 }
 
 function run_e2e_tests(){
```

Comment:
Curious if you want to enable tests from `./test/e2e/systeminternaltls` or what's the plan for those tests? I want to add the test for certificate rotation there.

Reply:
Let's see if that works out. We'll need patches for the tests, as OCP is a bit different. But we need to wait for https://github.com/knative/serving/pull/15217, otherwise we'll have to do it twice.